app/controllers/organizations/pets_controller.rb

```diff
@@ -31,9 +31,17 @@ def show
   def create
     @pet = Pet.new(pet_params)
 
-    if @pet.save
+    ActiveRecord::Base.transaction do
+      @pet.save!
+      Organizations::DefaultPetTaskService.new(@pet).create_tasks
```

Comment:
To make this work you'll need to make sure that `create_task` raises an error that is uncaught. I think the class we have it caught so doesn't bubble up here.

Reply:
Ahh yes right I see now.